test/specs/configs/empty.json

```diff
@@ -0,0 +1,25 @@
+{
+    "strains": {
+        "default": {
+            "code": "",
+            "condition": "",
+            "content": {
+                "filepath": "",
+                "owner": "local",
+                "ref": "master",
+                "repo": "default"
+            },
+            "directoryIndex": "index.html",
```

Comment:
should be `index`

Reply:
hmmm...ok.

Reply:
I think `directoryIndex` make it more clear. `index` could be anything.

Reply:
Then you have to change it in a number of places to make it consistent. I also prefer the shorter config parameters that don't get us in the camels vs. snakes discussion, so that's why I picked `index` in #103